pymapd/ipc.py

```
@@ -4,28 +4,33 @@
 import platform
 from ctypes.util import find_library
 
-# find c in OS
+# find c in OS, ipc not supported on Windows so return None
 libc_so = find_library('c')
-libc = ctypes.CDLL(libc_so)
+if platform.system() == "Windows":
```

Instead of wrapping this up here with the if/else, maybe instead we should just move all of this into the `load_buffer` function. 

If we're going to skip Windows entirely, then no need to even set the variables to None, we should just throw the assertion error to explain that its not supported on Windows. Then in the `else` clause, we can try the `find_library('c')` call and when we find it then load the buffer, otherwise throw the assertion error.

---

That's a good point as the code is a little awkward, the problem is these:
https://github.com/omnisci/pymapd/blob/2b5f4593676a2533f1c5c5fbc0155b85650ede9e/pymapd/connection.py#L27
https://github.com/omnisci/pymapd/blob/2b5f4593676a2533f1c5c5fbc0155b85650ede9e/pymapd/_parsers.py#L14
So we'd have to do some other changes in those files as well which seemed like a more radical code change.